entitydef/user-entitymodel_view.xml

```diff
@@ -0,0 +1,61 @@
+<entitymodel xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:noNamespaceSchemaLocation="https://ofbiz.apache.org/dtds/entitymodel.xsd">
+    <!-- ========================================================= -->
+    <!-- ======================== Defaults ======================= -->
+    <!-- ========================================================= -->
+    <title>View Entity of Custom UserMgr application</title>
+    <description>None</description>
+    <copyright></copyright>
+    <version></version>
+    
+    <view-entity entity-name="UserAndParty" package-name="org.apache.ofbiz.user">
+        <member-entity entity-alias="UL" entity-name="UserLogin"/>
+        <member-entity entity-alias="P" entity-name="Person"/>
+        <member-entity entity-alias="PCM" entity-name="PartyContactMech"/>
+        <member-entity entity-alias="PA" entity-name="PostalAddress"/>
+        <member-entity entity-alias="TN" entity-name="TelecomNumber"/>
+        <member-entity entity-alias="CM" entity-name="ContactMech"/>
+        <member-entity entity-alias="PCMP" entity-name="PartyContactMechPurpose"/>
+        <member-entity entity-alias="ULSG" entity-name="UserLoginSecurityGroup"/>
+        <alias name="userLoginId" entity-alias="UL" field="userLoginId" prim-key=""/>
```

Comment:
Why we use prim-key in view entity ?

Reply:
The fields that formed the primary key of the member entities are included in the primary key of the view-entity. The prim-key attribute is used to identify such fields(alias) explicitly. If not specified, the Entity Engine will examine the field's definition from which it originated(member-entity) to determine if it is part of the original entity's(member-entity's) primary key.

**Note:** There were discrepancies in the earlier commit of the view-entity "UserAndParty". In the recent commit "Created User Profile screen" those discrepancies have been eliminated.

Reply:
we don't have to add this prim-key tag to view entity

also when you add alias in the view entity do like this 
<alias entity-alias="UL" field="userLoginId" name="userLoginId"/>
so if the field and name is same then we do not need to add the "filed"  tag.

Reply:
Yes sir I have removed the field attribute from the alias tag wherever I am using the same alias name as of field.